apel/db/loader/loader.py

```diff
@@ -136,7 +136,7 @@ def load_all_msgs(self):
         while self.current_msg:
             if not self._inq.lock(self.current_msg):
                 log.warning("Skipping locked message %s", self.current_msg)
-                self.current_msg = self._inq.next()
+                self.current_msg = next(self._inq)
```

Comment:
ah okay, this is the Python 3 way of using next - see: https://peps.python.org/pep-3114/

Reply:
though would this break python2.7 support? Does `.next()` still exist in Python 3?

Maybe we need:

```
try:
    self.current_msg = next(self._inq)
except SomeException:
    self.current_msg = self._inq.next()
```

Reply:
I have added this in https://github.com/apel/apel/pull/354 https://github.com/apel/apel/commit/ef8d4b3312aee2ac17524ad1c566ff3856d64be3 and dd4bd0d3fe02d7a86c33f84c9f8beb117f8d2fa7.

This next() will be handled by future for cross-compatibility(dd4bd0d3fe02d7a86c33f84c9f8beb117f8d2fa7)